### PR TITLE
chore: add azure-pipelines CI/CD configuration

### PR DESCRIPTION
**Pull request by miurahr**
Add azure pipelines configuration when GitHub become a truth of the code.

## Pull request type

- Build and release changes -> [build/release]

## What does this PR change?

- Prepare for 6.1 development and future releases.

## Other information

TODO

- [ ] Integrate JRE Prep task

### CODE REVIEW

**Review by amake — APPROVED** [review]
I haven't tested anything but LGTM.

You probably don't want to make big changes in this PR, but in the future you could get rid of the JRE Prep task and download the JRE directly in the relevant task(s).

**Comment by miurahr** [comment]
> I haven't tested anything but LGTM.
> 
> You probably don't want to make big changes in this PR, but in the future you could get rid of the JRE Prep task and download the JRE directly in the relevant task(s).

Thank you for the suggestion.

File: ci/azure-pipelines/release_steps.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    inputs:
+      #  the name or unique identifier (GUID) of the secure file
+      # "jean-christophe_helary.pfx": '351b5d1f-5047-4d45-92a4-2384005085f7'
+      secureFile: "omegat_team.pfx"
+      retryCount: 5
+  - script: |
+      umask a+w
+      mkdir -p build
+      chmod -R a+w doc_src
+      $(Build.SourcesDirectory)/gradlew -PenvIsCi -PassetDir=$(System.ArtifactsDirectory)/asset/drop -PwinCodesignFile=$(cert.secureFilePath) -PwinCodesignTimestampUrl=http://timestamp.sectigo.com clean build manualPdfs
```
Comment (miurahr) [inline ci/azure-pipelines/release_steps.yml:22]:
```suggestion
      $(Build.SourcesDirectory)/gradlew -PenvIsCi -PassetDir=$(System.ArtifactsDirectory)/asset/drop clean build manualPdfs
```

File: ci/azure-pipelines/release_steps.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    displayName: 'Command Line Script'
+    env:
+      ORG_GRADLE_PROJECT_winCodesignPassword: $(keystorePassword)
+      ORG_GRADLE_PROJECT_jwsStorepass: $(keystorePassword)
+  - task: CopyFiles@2
```
Comment (miurahr) [inline ci/azure-pipelines/release_steps.yml:27]:
```suggestion
  - task: CopyFiles@2
```

**Review by miurahr — COMMENTED** [review]
We also need to change build.gradle not to try to sign on CI environment.

**Pull request merged**